tsconfig.json

```diff
@@ -2,9 +2,9 @@
   "compilerOptions": {
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
+    "target": "ES2020" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
```

Comment:
if the compile target is ES2020, does it still run in node versions that don't support `.matchAll()`?

Reply:
No, this is not just a syntactic sugar and it cannot be transpiled. That's why engine is defined to be 12 or more.
For a production lib, I wouldn't use it. But for this one? 🤷‍♂️